packages/darker/package.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: darker
+description: Apply black reformatting to Python files only in regions changed since a given commit.
+homepage: https://pypi.org/project/darker/
+licenses:
+  - BSD-3-Clause
+languages:
+  - Python
+categories:
+  - Formatter
+
+source:
+  id: pkg:pypi/darker@1.7.2?extra=isort,flynt
```

Comment:
I'm not actually sure if specifying multiple extras work. At least `python3 -m pip install darker[isort,flynt]==1.7.2` works.

Reply:
I think this should work!